### ISSUE TITLE
Can not zoom

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/talk-desktop#74]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Open the window on 4k
2. Try to zoom it like 120% or alike for improved readability

### Actual behaviour
There is no option for it, neither does Ctrl + Scroll nor Ctrl + `+` work.
Maybe it's okay and instead a setting ( #27 ) should be implemented, but that should then also be usable with known shortcuts

**Comment by SystemKeeper** [comment]
![image](https://user-images.githubusercontent.com/1580193/226957116-3b7e8d55-0e2c-4cf7-8c52-9ee457928731.png)

Works on Mac?

**Comment by nickvergessen** [comment]
Right, works when opening the menu with `Alt`

**Comment by ShGKme** [comment]
> Ctrl + `+` work.

Did you try Ctrl + SHIFT + `+` ? Without `SHIFT` it works as `CTRL + =`.

**Comment by nickvergessen** [comment]
That's totally unexpected for me. Is it configurable which we could adjust?
